content/docs/integrating-with-other-libraries.md

```diff
@@ -39,21 +39,21 @@ class SomePlugin extends React.Component {
 }
 ```
 
-Note that we defined both `componentDidMount` and `componentWillUnmount` [lifecycle methods](/docs/react-component.html#the-component-lifecycle). Many jQuery plugins attach event listeners to the DOM so it's important to detach them in `componentWillUnmount`. If the plugin does not provide a method for cleanup, you will probably have to provide your own, remembering to remove any event listeners the plugin registered to prevent memory leaks.
+Not edin ki, biz hem `componentDidMount` hem de `componentWillUnmount` [lifecycle methods](/docs/react-component.html#the-component-lifecycle)'i tanımladık. Birçok jQuery eklentisi, olay dinleyicilerini DOM'a ekler, bu nedenle onları `componentWillUnmount`'tan ayırmak önemlidir. Eğer eklenti düzeltme için bir yöntem sağlamıyorsa, muhtemelen hafıza sızıntılarını önlemek için eklentinin kaydettiği herhangi bir olay dinleyicisini kaldırmayı hatırlayarak kendi önleminizi almak zorunda olacaksınız. 
 
 ### Integrating with jQuery Chosen Plugin {#integrating-with-jquery-chosen-plugin}
 
-For a more concrete example of these concepts, let's write a minimal wrapper for the plugin [Chosen](https://harvesthq.github.io/chosen/), which augments `<select>` inputs.
+Bu kavramların daha kesin bir örneği için, `<select>` girdileri arttırılmıs [Chosen](https://harvesthq.github.io/chosen/) eklentisi için küçük bir sarmalayıcı yazalım.
```

Comment:
What about using "`<select>` girdilerini genişleten" in here?

Reply:
Both words have different meaning when you read the sentence, I would prefer to keep the existing one however, if you like to change the existing one, you could do so.

Reply:
Yes, the words are the same. But the current sentence is not natural. We need to fix that somehow.